packages/jest-runtime/src/script_transformer.js

```diff
@@ -20,7 +20,7 @@ import path from 'path';
 import vm from 'vm';
 import {createDirectory} from 'jest-util';
 import fs from 'graceful-fs';
-import {transform as babelTransform} from 'babel-core';
+import {transform as babelTransform} from '@babel/core';
```

Comment:
FYI, `babelTransform` may return `null` if the file being compiled has been `ignore`ed in Babel's config. It looks like this transformer doesn't handle that case. Looks like you'd want to `return content;` in that case.

Reply:
Didn't mean to resolve, sorry! But I think we handle it, see https://github.com/facebook/jest/blob/025c6af5e4bd2a971f295fe9408f3d8c77ad187b/packages/jest-runtime/src/script_transformer.js#L214-L234

Reply:
Submitting this again since GitHub's review system is still pretty wonky...

I think we handle it, see https://github.com/facebook/jest/blob/025c6af5e4bd2a971f295fe9408f3d8c77ad187b/packages/jest-runtime/src/script_transformer.js#L214-L234

We only assign to `transformed` if the transform returns something

Reply:
Oh you're right. The line I was worried about is https://github.com/facebook/jest/blob/025c6af5e4bd2a971f295fe9408f3d8c77ad187b/packages/jest-runtime/src/script_transformer.js#L155, but that actually shouldn't ever have any ignores anyway. There _is_ an issue with that call though, so I'll add a comment down there.